simpa/core/simulation_modules/optical_simulation_module/__init__.py

```diff
@@ -150,3 +150,15 @@ def run_forward_model(self,
                                          illumination_geometry=_device)
             fluence = results[Tags.DATA_FIELD_FLUENCE]
         return {Tags.DATA_FIELD_FLUENCE: fluence}
+
+    def get_additional_mcx_flags(self):
+        """
+        retrieves additional mcx flags defined within the global settings tag MCX_ADDITIONAL_FLAGS
+        :return: additional flags defined for mcx
+        """
+        cmd = []
+        if Tags.MCX_ADDITIONAL_FLAGS in self.global_settings:
+            for flag in self.global_settings.get(Tags.MCX_ADDITIONAL_FLAGS[0]):
```

Comment:
We usually use `in self.component_settings[Tags.ADDITIONAL_FLAGS]` since the settings class takes care of indexing the first element of the tuple